test/powershell/enginecore/CommandDiscovery.Tests.ps1

```diff
@@ -0,0 +1,105 @@
+﻿Describe "Command Discovery tests" -Tags "CI" {
+
+    BeforeAll {
+        setup -f testscript.ps1 -content "TestScript"
+		setup -f testscripp.ps1 -content "TestScripp"
```

Comment:
Fixed.

---
In reply to: [79886355](https://github.com/PowerShell/PowerShell/pull/2285#discussion_r79886355) [](ancestors = 79886355)